File: src/server/DeepSeatServer/src/main/kotlin/com/deepseat/server/DeepSeatServer/vo/Liked.kt
package com.deepseat.server.DeepSeatServer.vo

class Liked(likedID: Int, userID: String, docID: Int?, commentID: Int?) {
    var likedID: Int
    var userID: String
    var docID: Int?
    var commentID: Int?

    init {
        this.likedID = likedID
        this.userID = userID
        this.docID = docID
        this.commentID = commentID
    }

    constructor(userID: String, docID: Int? = null, commentID: Int? = null) : this(0, userID, docID, commentID)
}

File: src/app/iOS/DeepSeat/DeepSeat/Model/Liked.swift
//
//  Liked.swift
//  DeepSeat
//
//  Created by Jisoo Woo on 2022/05/20.
//

import Foundation

class Liked{
    var likedID: Int
    var userID: String
    var docID: Int?
    var commentID: Int?
    
    init(likedID: Int, userID: String, docID: Int?, commentID: Int?){
        self.likedID = likedID
        self.userID = userID
        self.docID = docID
        self.commentID = commentID
    }
    
}
